### DISCUSSION TITLE
Qt: how to find qmlls tool

### DISCUSSION THREAD

**Discussion by dsazonoff** [microsoft/vcpkg#34878] [Q&A] [answered]
Seems that qmlls tool is not compiled during qt6 build? I see that PR with fix was merged, but qmlls tool is still missing: https://github.com/microsoft/vcpkg/pull/30171

Environment: macOS, vcpkg (x64-osx triplet)

**Comment by Neumann-A** [comment]
Seems like qtlanguageserver needs to be added as a dependency to qtdeclarative.

**Reply by dsazonoff** [reply]
[qtdeclarative/portfile.cmake](https://github.com/microsoft/vcpkg/pull/30171/files#diff-ef90a647f70f151f454976b59376b0286c22b9f2ed267d6d4f5b6912be6343c3) contains qmlls - so it doesn't introduce dependency?

**Reply by Neumann-A** [reply]
no the vcpkg.json needs to declare that dependency.

**Reply by Neumann-A** [reply, marked as answer]
#34893 builds the executable now.
